cartridge-cli.lua

```diff
@@ -1877,6 +1952,71 @@ local function pack_deb(source_dir, dest_dir, name, release, version, opts)
     fio.copyfile(fio.pathjoin(tmpdir, deb_file_name), dest_dir)
 end
 
+local function pack_docker(source_dir, _, name, release, version, opts)
+    opts = opts or {}
+
+    local tmpdir = fio.tempdir()
+    print("Packing docker in: " .. tmpdir)
+
+    local distribution_dir = fio.pathjoin(tmpdir, name)
+    form_distribution_dir(source_dir, distribution_dir, name, version, {
+        skip_tarantool_binaries = true,
+        skip_build = true,
+    })
+
+    local expand_params = {
+        name = name,
+        instance_name = '${"$"}{TARANTOOL_INSTANCE_NAME:-default}',
+        workdir = fio.pathjoin('/var/lib/tarantool/', name),
+        dir = fio.pathjoin('/usr/share/tarantool/', name),
+    }
+
+    if tarantool_is_enterprise() then
+        local tnt_version_filepath = fio.pathjoin(get_tarantool_dir(), 'VERSION')
```

Comment:
This code should be moved to a function

Reply:
Also, probably the whole block of code that is responsible for generating the contents of the Dockerfile should be moved to a separate function, because it is clearly a separate concern and will make it easier to read and understand this code

Reply:
Moved `Dockerfile` creation to `construct_dockerfile` function.

Reply:
I think it will be more idiomatic if this function returns the string contents of the Dockerfile